echo-core/src/main/java/com/netflix/spinnaker/echo/notification/AbstractEventNotificationAgent.java

```diff
@@ -47,11 +47,8 @@ public abstract class AbstractEventNotificationAgent implements EventListener {
           .build();
 
   private static final Map<String, String> MANUAL_JUDGMENT_CONDITIONS =
-      ImmutableMap.<String, String>builder()
-          .put(ManualJudgmentCondition.MANUAL_JUDGMENT.getName(), StageStatus.STARTING.getName())
-          .put(ManualJudgmentCondition.CONTINUE.getName(), StageStatus.COMPLETE.getName())
-          .put(ManualJudgmentCondition.STOP.getName(), StageStatus.FAILED.getName())
-          .build();
+      ImmutableMap.of("manualJudgment", "starting");
```

Comment:
Be good to continue to use the enum!

Reply:
I revert that to use enum @jasonmcintosh